### PR TITLE
fix: #6 Make the collection capped with a default size of 10Gb

### PR DESCRIPTION
**Pull request by lukasz-gryzbon**

### CODE REVIEW

**Review by nickmcdowall — APPROVED** [review]
Hey - nice work 👍 

Couple questions: 

* I believe the `TTL` approach on indexes does not work with the capped collections - do we need to remove the `TTL` option from the index on the `createdAt` field?

* Also wondering if we should accept `MB` or `GB` instead of bytes to simplify the usage for clients looking to override the default capped collection (i.e. we convert for them into bytes)?

File: src/test/java/io/lsdconsulting/lsd/distributed/mongo/integration/InterceptedDocumentMongoRepositoryResiliencyIT.java [hunk 1]
```diff
@@ -45,20 +45,20 @@ static void tearDownTestClient() {
 
     @Test
     public void shouldHandleDbBeingDownGracefullyOnStartup() {
-        new InterceptedDocumentMongoRepository("mongodb://" + randomAlphabetic(10), DB_CONNECTION_TIMEOUT);
+        new InterceptedDocumentMongoRepository("mongodb://" + randomAlphabetic(10), DB_CONNECTION_TIMEOUT, 1024 * 100L);
```
[the remark this answers is not in the document]
Comment (lukasz-gryzbon) [reply src/test/java/io/lsdconsulting/lsd/distributed/mongo/integration/InterceptedDocumentMongoRepositoryResiliencyIT.java:48]: Good idea. Thanks

File: src/main/java/io/lsdconsulting/lsd/distributed/mongo/repository/InterceptedDocumentMongoRepository.java [hunk 2]
```diff
@@ -37,7 +38,8 @@
 @Slf4j
 public class InterceptedDocumentMongoRepository implements InterceptedDocumentRepository {
 
-    public static final int DEFAULT_TIMEOUT = 500;
+    public static final int DEFAULT_TIMEOUT_MILLIS = 500;
+    public static final long DEFAULT_COLLECTION_SIZE_LIMIT_BYTES = 1024 * 1000 * 1000 * 10L; // 10Gb
```
[the remark this answers is not in the document]
Comment (lukasz-gryzbon) [reply src/main/java/io/lsdconsulting/lsd/distributed/mongo/repository/InterceptedDocumentMongoRepository.java:42]: Yeah, I agree. Good idea.

**Comment by lukasz-gryzbon** [comment]
> Hey - nice work 👍
> 
> Couple questions:
> 
> * I believe the `TTL` approach on indexes does not work with the capped collections - do we need to remove the `TTL` option from the index on the `createdAt` field?
> * Also wondering if we should accept `MB` or `GB` instead of bytes to simplify the usage for clients looking to override the default capped collection (i.e. we convert for them into bytes)?

Very good points.

The expiry on index has been removed. Units changed to MBs.

Please have a look again. Thanks!

**Pull request merged**